tdrs-frontend/.snyk

```diff
@@ -1,69 +0,0 @@
-# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
```

Comment:
These ignores are all expired even if we choose to continue using Snyk on a different organization instead of accepting this PR's changes.